### DISCUSSION TITLE
Using "Duble Tab" for <winget install> command.

### DISCUSSION THREAD

**Discussion by furcanomer** [microsoft/winget-cli#832] [Ideas]
We should be able to use "Duble Tab" like Linux terminal commands.

Example:
sudo apt install chromiu... "Duble Tab" --> chromium

winget install msed... "Duble Tab" --> msedge

**Comment by denelon** [comment]
With PowerShell auto-complete enabled, you do get tab completion. 

https://github.com/microsoft/winget-cli/blob/master/doc/Completion.md
